src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -411,6 +424,7 @@ export default {
 			this.diffInDays = differenceInCalendarDays(parseISO(loan?.plannedExpirationDate), new Date());
 			this.hasThreeDaysOrLessLeft = this.diffInDays <= 3;
 			this.lender = result?.data?.my?.userAccount ?? {};
+			this.shownModal = !this.cookieStore.get('what-is-kiva-shown') && !result?.data?.hasEverLoggedIn;
```

Comment:
Shouldn't this be `this.cookieStore.get('what-is-kiva-shown') || result?.data?.hasEverLoggedIn`?